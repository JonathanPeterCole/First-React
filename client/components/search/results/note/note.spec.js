import React from 'react'
import { shallow } from 'enzyme'

// Component
import Note from './note'

// Tests
it('Notes render correctly', () => {
  const wrapper = shallow(<Note title='Test Title' body='Test Body' />)
  expect(wrapper).toMatchSnapshot()
})
