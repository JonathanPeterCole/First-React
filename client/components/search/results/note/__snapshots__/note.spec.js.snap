// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notes render correctly 1`] = `
<div
  className="note"
>
  <div
    className="title"
  >
    Test Title
  </div>
  <div
    className="body"
  >
    Test Body
  </div>
</div>
`;
